test/test_model.py

```diff
@@ -205,11 +202,263 @@ def test_model_details_outputted_correctly(self, mock_click):
             # ASSERT
             assert mock_click.echo.call_args_list == [
                 call(
-                    "Name: display name     ID: version id     Date: 03/03/2021 00:00:00"
+                    "Name: display name"
+                    + TAB_SPACE
+                    + "ID: version id"
+                    + TAB_SPACE
+                    + "Date: March 03 2021"
                 ),
                 call("Summary: summary"),
             ]
 
+        @patch("dafni_cli.model.prose_print")
+        def test_model_details_outputted_correctly_with_description(
```

Comment:
I'd probably suggest updating the name to mention its because your passing in long=True